Sources/BuildSystemIntegration/SwiftPMBuildSystem.swift

```diff
@@ -291,11 +280,11 @@ package actor SwiftPMBuildSystem: BuiltInBuildSystem {
     self.testHooks = testHooks
     self.connectionToSourceKitLSP = connectionToSourceKitLSP
 
-    guard let destinationToolchainBinDir = toolchain.swiftc?.parentDirectory else {
+    guard let destinationToolchainBinDir = toolchain.swiftc?.deletingLastPathComponent() else {
       throw Error.cannotDetermineHostToolchain
     }
 
-    let hostSDK = try SwiftSDK.hostSwiftSDK(AbsolutePath(destinationToolchainBinDir))
+    let hostSDK = try SwiftSDK.hostSwiftSDK(try AbsolutePath(validating: destinationToolchainBinDir.filePath))
```

Comment:
Is the second `try` needed?

Reply:
No, probably an artifact of the refactoring. Thanks for spotting.